sw/device/silicon_creator/lib/test_main.h

```diff
@@ -6,7 +6,7 @@
 #define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_TEST_MAIN_H_
 
 #include "sw/device/lib/runtime/log.h"
-#include "sw/device/lib/testing/test_framework/test_main.h"
+#include "sw/device/lib/testing/test_framework/ottf.h"
```

Comment:
This file (`sw/device/silicon_creator/lib/test_main.h`) is used by mask rom functests. Can you also update the `mask_rom_tests` `foreach` loop in `sw/device/silicon_creator/testing/meson.build`?

Reply:
Ah, my bad, missed that. Fixed.